packages/react/src/_popover/Popover.tsx

```diff
@@ -161,10 +153,16 @@ export function usePopover<RT extends ReferenceType = ReferenceType>(
 		};
 	}
 
-	function getPopoverProps() {
+	function getPopoverProps(isOpen?: boolean) {
 		return {
 			ref: floating.refs.setFloating,
-			style: floating.floatingStyles,
+			style: {
+				...floating.floatingStyles,
+				// For conditional rendering to work, we always render the Popover
+				// and only conditionally render the content. Hiding these borders
+				// when the Popover is closed prevents artefacts remaining in the UI.
+				...(isOpen && { borderLeftWidth: 0, borderRightWidth: 0 }),
```

Comment:
Why are you removing the borders when it's open? It looks weird now 
<img width="774" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/28ba6c02-7d70-4d1d-a514-2617fa643d48">

Reply:
Oh, it's supposed to be when it's closed. Did you know that Popover has a `visibility` prop that we can probably use instead?

Reply:
Oops, I'll fix that up.

I tried a bunch of different things and IIRC `visibility` caused the same original issue. I'll take a look, though.

- [ ] Look at `visibility` prop
- [ ] If ☝️ doesn't work, switch this to `!isOpen`